### ISSUE TITLE
The Title Bar ignores dark mode

### ISSUE THREAD

**Issue by drocx** [getinsomnia/insomnia#3150]
**Describe the bug**
The Title Bar Color is white on Windows 10 with dark mode enabled.

**To Reproduce**
1. Switch to windows dark mode
2. Start Insomnia

**Expected behavior**
The Title Bar should use the dark mode like vs code or other electron apps.

**Screenshots**
![image](https://user-images.githubusercontent.com/2034544/110118253-5c3fb480-7dba-11eb-97ec-640ac7daacbe.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 20H2 Build 19042.844
 - App Version: 2021.1.0

**Comment by nijikokun** [comment]
This is definitely a good first issue for someone who wants to get into open source, looks like we are missing or overriding an electron setting:

```
const { nativeTheme } = require('electron')
nativeTheme.themeSource = 'system'
```

**Comment by anyforever** [comment]
i want to hide it !!!

**Comment by Warkanlock** [comment]
On it!

**Comment by roguib** [comment]
It seems, for this particular [issue](https://github.com/electron/electron/issues/23479) that ```nativeTheme.themeSource = 'system'``` doesn't change the color of the title bar in the current electron version of the project. It hasn't been until [this PR](https://github.com/electron/electron/pull/25373) was merged that the issue has been partially solved. 

I've tried setting ```nativeTheme.themeSource = 'system'``` on different places suggested and it didn't change the color of the title bar to match the dark mode of the system. I've been researching a bit and it seems there're several workarounds to this issue such as implementing your custom title bar, use a complicated API to set the color of the bar or upgrade Electron.
